src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodRepository.scala

```diff
@@ -62,9 +62,9 @@ object MethodRepository {
     val ListAll = List("All")
   }
 
-  def validateEmail(email:String): Boolean = {
+  def validatePublicOrEmail(email:String): Boolean = {
     // TODO: real email address validation!
-    !email.trim.isEmpty
+    email.trim.equalsIgnoreCase("public") || !email.trim.isEmpty
```

Comment:
isn't `!email.trim.isEmpty` a superset of `email.trim.equalsIgnoreCase("public")`?